Buildings/Controls/OBC/Utilities/PIDWithAutotuning/FirstOrderAMIGO.mo

```diff
@@ -387,7 +387,7 @@ This block implements a rule-based tuning method for a PI or a PID controller, w
 steps:
 </p>
 <p>
-Step 1: Introducing a periodic disturbance
+Step 1: Introducing a relay disturbance
```

Comment:
`Introducing` or `Introduce`? I see in below section, `Step 1: Conducting a &quot;test run&quot;` is changed to `Step 1: Conduct a &quot;test run&quot;`.